manifests/plugins/signalfx.pp

```diff
@@ -22,11 +21,11 @@
   collectd::check_and_install_package { 'signalfx-collectd-plugin':
     before  => File['load Signalfx plugin']
   }
-  if $::osfamily == 'Redhat' {
```

Comment:
No need to remove the os check. The collectd-python plugin is bundled in the SignalFx Collectd Package for ubuntu/debian.  This causes apt-get to try to install collectd-python package which then fails.